runsc/cgroup/cgroup_test.go

```diff
@@ -25,6 +25,83 @@ import (
 	"gvisor.dev/gvisor/pkg/test/testutil"
 )
 
+var debianMountinfo = `
```

Comment:
It's great to use real example for tests, but in order to make it easier to understand, let's cut down the data to keep only the interesting test cases, e.g. cgroup and one no cgroup file systems, options can be removed to reduce clutter, add a mount info that is malformed to test error cases, etc.

Reply:
done.